qt/python/mantidqt/widgets/tableworkspacedisplay/presenter.py

```diff
@@ -29,6 +30,14 @@ class TableWorkspaceDisplay(object):
     TOO_MANY_SELECTED_FOR_PLOT = "Too many columns are selected to plot. Please select only 1."
     NUM_SELECTED_FOR_CONFIRMATION = 10
     NO_COLUMN_MARKED_AS_X = "No columns marked as X."
+    ITEM_CHANGED_INVALID_DATA_MESSAGE = "Error: Trying to set invalid data for the column."
+    ITEM_CHANGED_UNKNOWN_ERROR_MESSAGE = "Unknown error occurred: {}"
+    TOO_MANY_TO_SET_AS_Y_ERR_MESSAGE = "Too many selected to set as Y Error"
+    CANNOT_PLOT_AGAINST_SELF_MESSAGE = "Cannot plot column against itself."
+    NO_ASSOCIATED_YERR_FOR_EACH_Y_MESSAGE = "There is no associated YErr for each selected Y column."
+    PLOT_FUNCTION_ERROR_MESSAGE = "One or more of the columns being plotted contain invalid data for MatPlotLib.\n\nError message:\n{}"
```

Comment:
The casing of `MatPlotLib` is a little strange - can we use just `Matplotlib`? It's what they themselves use: https://matplotlib.org/.